.github/actions/setup-env/action.yml

```diff
@@ -21,6 +21,13 @@ inputs:
 runs:
   using: "composite"
   steps:
+    # Setup dev-env
+    - name: Setup ov
+      shell: bash
+      run: |
+        ./ov build
+        ./ov setup-env
```

Comment:
I would have thought this would need to include a call to `ov just install` as well; or is the plan to install specifically for each CI step? If the whole `ov` container and volume is going to be cached between jobs then installing once at the start would save some time on subsequent jobs :thinking:. I don't know if the volume will work that way, though.